pkg/server/routes.go

```diff
@@ -34,11 +35,45 @@ type ApiResponse struct {
 	Data interface{} `json:"data"`
 }
 
-func respondWithJson(w http.ResponseWriter, obj interface{}) {
-	resp := ApiResponse{Data: obj}
-	err := json.NewEncoder(w).Encode(resp)
-	if err != nil {
-		fmt.Fprint(w, err)
+type NotificationStatus int
+
+const (
+	SUCCESS NotificationStatus = 0
+	RUNNING NotificationStatus = 1
+	FAILURE NotificationStatus = 2
+)
+
+func (ns NotificationStatus) String() string {
+	values := [...]string{
+		"SUCCESS",
+		"RUNNING",
+		"FAILURE",
+	}
+	if ns < SUCCESS || ns > FAILURE {
+		return "Unknown"
+	}
+
+	return values[ns]
+}
+
+func (ns NotificationStatus) MarshalJSON() ([]byte, error) {
+	return json.Marshal(ns.String())
+}
+
+type Notification struct {
```

Comment:
At some point, we'll need to figure out standardized notification format, probably aligned to day2 (I expect 'type' to identify the message and 'payload' for additional data)